### DISCUSSION TITLE
Custom UI scaling

### DISCUSSION THREAD

**Discussion by psychoanima** [opentoonz/opentoonz#6833] [General]
### Ask a question related to OpenToonz

Hello all,

I just install OT on 10" Surface tablet. Resolution of the screen is 1920 x 1280, but UI looks enormous.

<img width="960" alt="OT UI" src="https://user-images.githubusercontent.com/18187104/169048035-2bcfcb24-60f9-43d4-9b62-dcb7a07aac21.png">

Since it's Windows 11 OS I tried to change DPI trough properties (right click on OT icon, then high DPI settings in Compatibility tab ), but without success. Only way to do it is trough Windows System settings and to change global DPI from 150% to 100%. But then everything in WIndows is scaled down and unberable for the eyes on 10" screen.

Is there any custom UI scaling inside OT? If not, maybe a feature request?

**Comment by mrlizhaozhi** [comment]
It's now August 2023, and this issue persists in OpenToonz 1.7.1. I use a Lenovo Yoga laptop with the Windows 10 operating system. The window components are so large that some texts are squeezed into unreadable thin lines. 

After some trials and errors, (Properties -> Advanced DPI configuration -> High DPI scaling override -> System) saved my life. Although the texts seem to have downgraded resolution after this change, I haven't found any better workaround than DPI configuration.
